Add EWS-GetMessageTraceV2 (#40842)
demisto/content: Packs/MicrosoftExchangeOnline/ReleaseNotes/1_6_21.md

#### Integrations

##### EWS Extension Online Powershell v3

> Fixed ***ews-message-trace-get*** parameter config bug where *from_ip* and *to_ip* are treated as array arguments. 

Reviewer: ```suggestion
Fixed the ***ews-message-trace-get*** parameter where *from_ip* and *to_ip* were treated as array arguments. 
```

> Add ***ews-message-trace-get-v2*** command using Get-MessageTraceV2 Powershell cmdlet.

Reviewer: ```suggestion
Added the ***ews-message-trace-get-v2*** command using the Get-MessageTraceV2 PowerShell cmdlet.
```
